src/realizations/coastal/SchismCreator.cpp

```diff
@@ -53,12 +54,25 @@ std::unique_ptr<CoastalFormulation>
                       std::chrono::system_clock::from_time_t(start_time_t),
                       std::chrono::system_clock::from_time_t(stop_time_t));
 
+      SchismFormulation::check_forcing_provider( *netcdf_streamflow_provider, 
+		                                           SchismFormulation::CHANNEL_FLOW );
+
+      auto netcdf_offshore_provider = 
+	      std::make_shared<data_access::NetCDFMeshPointsDataProvider< 
+	                                                   data_access::TidalMeshPolicy > >(
+		      offshore_boundary_file,
+                      std::chrono::system_clock::from_time_t(start_time_t),
+                      std::chrono::system_clock::from_time_t(stop_time_t));
+
+      SchismFormulation::check_forcing_provider( *netcdf_offshore_provider, 
+		                                           SchismFormulation::OFFSHORE );
       return std::make_unique<SchismFormulation>( model_id,
                                             library_file,
 					    init_config,
                                             MPI_COMM_SELF,
 					    netcdf_met_provider, 
-                                            provider,
+                                            //provider,
+					    netcdf_offshore_provider, 
 					    netcdf_streamflow_provider
```

Comment:
Again, indentation is all over the place.

Reply:
Replaces tabs with spaces.